client-src/elements/chromedash-drawer.js

```diff
@@ -160,36 +163,36 @@ export class ChromedashDrawer extends LitElement {
   }
 
   initializeTestingSignIn() {
-    if (this.devMode != 'True') {
-      return;
-    }
-
     // Create DEV_MODE login button for testing
     const signInTestingButton = document.createElement('button');
     signInTestingButton.innerText = 'Sign in as example@chromium.org';
+    signInTestingButton.setAttribute('type', 'button');
```

Comment:
No need to do it in this PR, but it occurs to me that this testing button could be created using a lit-html approach like the way that regular product UI elements are created.

Reply:
Yes, it could, and probably should.  It was good to start simple since there were so many unknowns.  

By the way, I will also probably eliminate the redundant screenshot checking for the login before each test, and perhaps figure out how to login just once for a set of tests.